dipy/workflows/tracking.py

```diff
@@ -18,6 +20,58 @@
 from dipy.data import default_sphere
 from dipy.direction import DeterministicMaximumDirectionGetter
 from dipy.workflows.multi_io import io_iterator_
+from ipdb import set_trace
+
+
+def det_track_flow(peaks_files, stopping_files, use_sh=False,
```

Comment:
how would you best handle tracking from a sh file (e.g. csd_sh.nii.gz)? an other flow?